### PR TITLE
176666940 sorting tray updates

### PR DESCRIPTION
**Pull request by mklewandowski**
This PR populates the sorting tray with animal images from the animal populations created when the sim is run.
- remove obsolete animals
- add animal sensitivity levels
- add animal SVGs
- populate tray with animal SVGs when sim run is complete, random locations and rotations
- improve tray layout to match UI spec
- updated text translations, include tray header in translation  (note: final Spanish translation will be done in future work)
- hide Tray when user rewinds sim

image

### CODE REVIEW

File: src/components/simulation/tray.tsx [hunk 1]
```diff
@@ -14,27 +16,26 @@ export const Tray: React.FC<IProps> = (props) => {
   const { trayAnimals, hidden, onHideTray } = props;
   return (
     <div className={`tray ${hidden ? "hidden" : ""}`}>
+      <SortingTray />
       <div className="header">
-        <div className="title">Leaf Pack Sorting Tray</div>
+        <div className="title">{t("SORTINGTRAY")}</div>
         <button className="close" onClick={onHideTray}>
           <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12" width="12" height="12">
```
Comment (eireland) [inline src/components/simulation/tray.tsx:23]: Why is the close button svg here instead of an image folder?

Comment (mklewandowski) [reply src/components/simulation/tray.tsx:23]: It was easy to inline and I was following the DESE model that the orange team was using for these simple SVGs (they do the same thing with the thumbnail x close).  Probably worth having in an asset eventually - especially if we need to reuse it.

Comment (mklewandowski) [reply src/components/simulation/tray.tsx]: I moved it into a file.  Now we can reuse it anywhere using the same pattern that we have for the other SVGs.

**Review by eireland — APPROVED** [review]
Looks good!

**Pull request merged**